tools/st2_deploy.sh

```diff
@@ -298,7 +298,7 @@ start on runlevel [2345]
 stop on runlevel [016]
 respawn
 
-exec /opt/openstack/mistral/.venv/bin/python /opt/openstack/mistral/mistral/cmd/launch.py --config-file /etc/mistral/mistral.conf --log-config-append /etc/mistral/wf_trace_logging.conf
+exec /opt/openstack/mistral/.venv/bin/python /opt/openstack/mistral/mistral/cmd/launch.py --config-file /etc/mistral/mistral.conf --log-file /var/log/mistral.log --log-config-append /etc/mistral/wf_trace_logging.conf
```

Comment:
Is it? There are three ways this is represented then. One with flags, different between RHEL/Debian, and then a config file. The test should be canonical. What do we want?